code/game/gamemodes/cult/runes.dm

```diff
@@ -780,6 +780,11 @@ var/list/teleport_runes = list()
 		fail_invoke()
 		log_game("Summon Cultist rune failed - target in away mission")
 		return
+	if(!isliving(cultist_to_summon))
```

Comment:
god I knew it wouldn't be that simple, well at least I opened it before the feature freeze